README.md

```diff
@@ -116,6 +116,15 @@ vars:
     microsoft_ads_<default_source_table_name>_identifier: your_table_name 
 ```
 
+#### Change how ad name is determined
```

Comment:
can you move this farther up, like before the `Change the build schema` section?

Reply:
done!